presto-main/src/main/java/io/prestosql/operator/LookupJoinPageBuilder.java

```diff
@@ -179,11 +181,23 @@ private void appendProbeIndex(JoinProbe probe)
         if (previousPosition == position) {
             return;
         }
+        estimatedProbeBlockBytes += getEstimatedProbeRowSize(probe);
+    }
+
+    private int getEstimatedProbeRowSize(JoinProbe probe)
+    {
+        if (estimatedProbeRowSize != -1) {
+            return estimatedProbeRowSize;
```

Comment:
Can you great greatly off estimates when initial data is outlier?

Reply:
the estimate per row is always the same. It was always the same previously too.

`block.getSizeInBytes() / block.getPositionCount();` takes into account entire block size

Reply:
i missed the reset call, thanks